.vscode/launch.json

```diff
@@ -1,6 +1,20 @@
 {
   "version": "0.2.0",
   "configurations": [
+    {
```

Comment:
Gah, let me remove this. …and then wait for CI to run for another 20 minutes. «sigh»